### ISSUE TITLE
Mac brush resource selector

### ISSUE THREAD

**Issue by ermau** [xamarin/Xamarin.PropertyEditing#344]
- [ ] Theming - pending #334
- [ ] Filtering

![](https://user-images.githubusercontent.com/4982/35685247-701174a0-0769-11e8-8135-181cd125230b.png)

We'll drop the bottom +/- bar since we don't currently support free-form resource creation.

**Comment by lewing** [comment]
This is for the column layout and the filter search box?

**Comment by lewing** [comment]
@vancura is that resource filter icon available anywhere?

**Comment by vancura** [comment]
Yup, https://github.com/xamarin/icons/blob/master/property-pad/property-brush-resources-16%402x.png (and the rest of `property-brush-resources-*.png` icons in that directory.

**Comment by lewing** [comment]
I meant 
![image](https://user-images.githubusercontent.com/24063/42972309-a2aa1e0a-8b74-11e8-96da-ef55c7b32728.png)

**Comment by vancura** [comment]
Here it is: http://vncr.in/t6qh
